.github/workflows/host_core.yml

```diff
@@ -13,22 +13,35 @@ env:
 jobs:
   build:
     strategy:
+      fail-fast: false
       matrix:
+        os: [ubuntu-20.04, windows-2019, macos-10.15]
```

Comment:
I picked versions specifically just to avoid getting upgraded on by Github, for example windows-latest was 2019 until just a bit ago when they upgraded to `windows-2022`